### PR TITLE
docs: update 2021-12 and 2022/02 blog's desc/keywords/tags

### PR DESCRIPTION
**Pull request by guitu168**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: blog/en/blog/2021/12/07/apisix-integrate-skywalking-plugin.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ authors:
...
 keywords: 
 - Apache APISIX
+- API Gateway
 - Apache SkyWalking
 - Log Process
 - Plugin
-description: This paper mainly introduces two Apache APISIX integrated SkyWalking log plug-ins to provide a more convenient operation and environment for log processing in Apache APISIX.
-tags: [Technology]
+description: This paper mainly introduces two Apache APISIX integrated SkyWalking log plugins to provide a more convenient operation and environment for log processing in Apache APISIX.
+tags: [Plugins]
```
Comment (SylviaBABY) [inline blog/en/blog/2021/12/07/apisix-integrate-skywalking-plugin.md]:
```suggestion
tags: [Plugins,Ecosystem]
```

File: blog/en/blog/2022/02/17/weekly-report-0214.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+- API Gateway
 - Weekly Report
 - Contributor
-- APISIX
-- API Gateway
+- OpenTelemetry
 - Apache
-description: The Apache APISIX Community Weekly Report hopes to help community members better understand the weekly progress of the Apache APISIX community and and facilitate everyone to participate in the Apache APISIX community.
-tags: [Events]
+description: The cloud-native API gateway Apache APISIX has supported plugins such as csrf, file-logger, public-api, opentelemetry, and loggly in the past two weeks.
```
Comment (SylviaBABY) [inline blog/en/blog/2022/02/17/weekly-report-0214.md]:
```suggestion
description: The cloud-native API gateway Apache APISIX has supported plugins such as csrf, file-logger, public-api, opentelemetry, and logging in the past two weeks.
```

File: blog/en/blog/2022/02/25/consul-api-gateway.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ keywords:
 - Consul
 - Service Discovery
 - Servici Register
-description: Apache APISIX supports the Consul KV-based service discovery registry. This article will walk you through the process of implementing service discovery and service registry in Apache APISIX.
-tags: [Technology,Ecosystem,Service Discovery]
+description: Apache APISIX supports the Consul KV-based service discovery registry. This article will walk you through the process of implementing service discovery and service registry in APISIX.
+tags: [Ecosystem,Service Discovery]
```
Comment (SylviaBABY) [inline blog/en/blog/2022/02/25/consul-api-gateway.md]:
```suggestion
tags: [Ecosystem]
```

File: blog/zh/blog/2021/12/07/apisix-integrate-skywalking-plugin.md [hunk 4]
```diff
@@ -11,11 +11,12 @@ authors:
...
 keywords: 
 - Apache APISIX
+- API 网关
 - Apache SkyWalking
 - 日志处理
 - 插件集成
-description: 本文主要介绍了两款 Apache APISIX 集成 SkyWalking 的日志插件，为之后大家在 Apache APISIX 中进行日志处理提供更方便的操作与环境。
-tags: [Technology]
+description: 本文主要介绍云原生 API 网关 Apache APISIX 集成 SkyWalking 的日志插件，为之后用户在 Apache APISIX 中进行日志处理提供更方便的操作与环境。
+tags: [Plugins]
```
Comment (SylviaBABY) [inline blog/zh/blog/2021/12/07/apisix-integrate-skywalking-plugin.md]:
```suggestion
tags: [Plugins,Ecosystem]
```

File: blog/zh/blog/2021/12/24/apisix-integrate-openwhisk-plugin.md [hunk 5]
```diff
@@ -11,11 +11,12 @@ authors:
...
 keywords: 
 - Apache APISIX
+- API 网关
 - Apache OpenWhisk
 - Serverless
 - 插件
-description: 本文为大家介绍了关于 `openwhisk` 插件的功能前瞻与使用步骤，结合 Apache APISIX 提供的多种身份认证插件来实现认证与授权等功能。
-tags: [Technology]
+description: 本文为大家介绍了关于 `openwhisk` 插件的功能前瞻与使用步骤，结合 API 网关 Apache APISIX 提供的多种身份认证插件来实现认证与授权等功能。
+tags: [Plugins]
```
Comment (SylviaBABY) [inline blog/zh/blog/2021/12/24/apisix-integrate-openwhisk-plugin.md]:
```suggestion
tags: [Plugins,Ecosystem]
```

File: blog/zh/blog/2022/02/25/consul-api-gateway.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ keywords:
 - Consul
 - 服务发现
 - 服务注册
-description: Apache APISIX 支持基于 Consul KV 的服务发现注册表。这篇文章讲述了在 Apache APISIX 中实现服务发现和服务注册的全过程。
+description: 云原生 API 网关 Apache APISIX 支持基于 Consul KV 的服务发现。本文讲述了在 Apache APISIX 中实现服务发现和服务注册的全过程及 consul_kv 的相关原理。
 tags: [Technology,Ecosystem,Service Discovery]
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/02/25/consul-api-gateway.md]: should be Ecosystem

**Review by SylviaBABY — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**